### ISSUE TITLE
Overview selection should be togglable

### ISSUE THREAD

**Issue by kaimikael** [SmartZoos/Toolset#129]
We have an icon that zooms out the current view so that one gets all activity items in view.  It would make sense if pressing the icon again brought the view back to the previous zoom level, so that one can get a quick glance at the whole and then return to the immediate surroundings.

**Comment by pjotrsavitski** [comment]
That is doable. The data about current position and zoom level could easily be captured and stored temporarily. Yet it would require the icon to be changed to look somehow different to signal the user what would happen. It should also be considered how long would that be available, going back. Consider a scenario where user presses the button, the location is stored. Then user does some other things. Presses the button again, yet the functionality takes the user to the previous center position of the map and also changes the zoom level.

This enhancement should at least have an additional icon, so that the user would understand what to expect.

**Comment by kaimikael** [comment]
I don't think it should reset the position, just the zoom factor.  But yes, it would make sense to have a two-state icon.
I suggest that if the player has manually changed the zoom factor, that should reset the base zoom factor.

**Comment by pjotrsavitski** [comment]
The current code will reset the position. The logic of the current solution is. Make sure that all the markers of the game are seen on the screen. The center of the map is no longer a user, but coordinates based on current set of points.

I could see if we could do that without changing the center of the map, but that is the way it works right now.
